tests/IResearch/IResearchDocument-test.cpp

```diff
@@ -1681,7 +1681,7 @@ TEST_F(IResearchDocumentTest, FieldIterator_nullptr_analyzer) {
                                        EMPTY, EMPTY, EMPTY,
                                        arangodb::transaction::Options());
 
-    arangodb::iresearch::FieldIterator it(trx);
```

Comment:
let's add a test check `FieldIterator`'s behavior with `_id` field when a collection argument is filled/not filled